src/Container/Tarball.hs

```diff
@@ -54,7 +54,7 @@ data TarEntries = TarEntries
 
 -- | Parses Container Image from Tarball Byte string.
 parse :: ByteStringLazy.ByteString -> Either (NLE.NonEmpty ContainerImgParsingError) ContainerImageRaw
-parse content = case mkEntries $ Tar.read content of
+parse content = case mkEntries $ Tar.read' content of
```

Comment:
We also use `Tar.read` on Archive extractor for unarchiver - which I did not change. 

Errors we are seeing when extracting archives in recursive unarchiving, may or may not solved by this (I believe there was a ticket in backlog for this). They may have also been due to filepath limit (since haskell/tar does not support PAX tar file fully).